lua/tint.lua

```diff
@@ -158,6 +158,11 @@ local function verify_version()
 end
 
 __.on_focus = function()
+  local winid = vim.api.nvim_get_current_win()
```

Comment:
Can we pass this through to the function below so we don't end up calling `nvim_get_current_win` twice?

Reply:
done 👍🏿